index.js

```diff
@@ -114,31 +112,32 @@ module.exports = function (spec) {
     if (results.tests.length === 0) {
       return pad(format.red(symbols.cross + ' No tests found'));
     }
-
-    return _.filter([
+ 
+    const temp = [
       pad('total:     ' + results.asserts.length),
       pad(format.green('passing:   ' + results.pass.length)),
       results.fail.length > 0 ? pad(format.red('failing:   ' + results.fail.length)) : undefined,
       pad('duration:  ' + prettyMs(new Date().getTime() - startTime))
-    ], _.identity).join('\n');
+    ];
+    return temp.join('\n');
```

Comment:
Sure, sorry! It's minor, but the `temp` array may look like:

```js
["total: <number>", "passing: <number>", undefined, "duration: <number>"]
```

If the `undefined` is not removed, the result of `join('\n')` may be:

```sh
total: <number>
passing: <number>

duration: <number>
```

One way to remove the `undefined` value before the `join` is to use `.filter()`